storage/demo/src/main/java/de/grobox/storagebackuptester/restore/DemoSnapshotFragment.kt

```diff
@@ -31,7 +31,7 @@ class DemoSnapshotFragment : SnapshotFragment() {
         bottomStub.layoutResource = R.layout.footer_snapshot
         val footer = bottomStub.inflate()
         footer.findViewById<Button>(R.id.button).setOnClickListener {
-            requireActivity().onBackPressed()
+            requireActivity().onBackPressedDispatcher.onBackPressed()
```

Comment:
If you change this for the demo app, don't you need to add `android:enableOnBackInvokedCallback="true"` to its manifest as well?

Reply:
Done